distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+|Path to server keystore
+|etc/keystores/serverKeystore.jks
+|Yes
+
+|Keystore Password
+|javax.net.ssl.keyStorePassword
+|String
+|Password for accessing keystore
+|changeit
+|Yes
+
+|Truststore
+|Truststorejavax.net.ssl.trustStore
+|String
+|The trust store used for SSL/TLS connections. Path is relative to `<INSTALL_HOME>`.
+|etc/keystores/serverTruststore.jks
+|Yes
+
+|Truststore Password
+|javax.net.ssl.trustStorePassword
+|String
+|Password for server Truststore
+|changeit
+|Yes
+
+|Keystore Type
+|javax.net.ssl.keyStoreType
+|String
+|File extension to use with server keystore
+|jks
+|Yes
+
+|Truststore Type
+|javax.net.ssl.trustStoreType
+|String
+|File extension to use with server truststore
+|jks
+|Yes
+
+6+^h| Global URL Properties
+
+|Protocol
+|org.codice.ddf.system.protocol
+|String
+|Default protocol that should be used to connect to this machine.
+|https://
+|Yes
+
+|Hostname
+|org.codice.ddf.system.hostname
+|String
+|The hostname or IP address used to advertise the system. Do not enter `localhost`. Possibilities include the address of a single node or that of a load balancer in a multi-node deployment.
+
+NOTE: Does not change the address the system runs on.
+|localhost
+|Yes
+
+|HTTPS Port
+|org.codice.ddf.system.httpsPort
+|String
+|The https port used by the system.
+
+NOTE: This *DOES* change the port the system runs on.
+|8993
+|Yes
+
+|HTTP Port
+|org.codice.ddf.system.httpPort
+|String
+|The http port used by the system.
+
+NOTE: This *DOES* change the port the system runs on.
+|8181
+|Yes
+
+|Default Port
+|org.codice.ddf.system.port
+|String
+|The default port used to advertise the system. This should match either the http or https port.
+
+NOTE: Does not change the port the system runs on.
+|8993
+|Yes
+
+|Root Context
+|org.codice.ddf.system.rootContext
+|String
+|The the base or root context that services will be made available under.
+|/services
+|Yes
+
+6+^h|System Information Properties
+
+|Site Name
+|org.codice.ddf.system.id
+|String
+|The site name for ${branding}.
+|ddf.distribution
+|Yes
+
+|Site Contact
+|org.codice.ddf.system.siteContact
+|String
+|The email address of the site contact.
+|
+|No
+
+|Version
+|org.codice.ddf.system.version
+|String
+|The version of ${branding} that is running.
+
+This value should not be changed from the factory default.
+|${project.version}
+|Yes
+
+|Organization
+|org.codice.ddf.system.organization
+|String
+|The organization responsible for this installation of ${branding}.
+|Codice Foundation
+|Yes
+
+6+^h|Thread Pool Settings
+
+|Thread Pool Size
+|org.codice.ddf.system.threadPoolSize
+|Integer
+|Size of thread pool used for handling UI queries, federating requests, and downloading resources
+|128
+|Yes
+
+6+^h|HTTPS Specific Settings
+
+|Cipher Suites
+|https.cipherSuites
+|String
+|Cipher suites to use with secure sockets
+|TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
+
+TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
+
+TLS_DHE_DSS_WITH_AES_128_CBC_SHA,
+
+TLS_RSA_WITH_AES_128_CBC_SHA
+|No
+
```

Comment:
We're missing this one: `jdk.tls.ephemeralDHKeySize=matched`. @stustison, can you provide a short description for this?